### PR TITLE
docs: ZENKOIO-116 small updates

### PR DESCRIPTION
**Pull request by dashagurova**
This fixes ZENKOIO-116

Started with several small updates to CloudServer documentation like:
- fixed broken links that led nowhere before or to a wrong place
- removed old badges for old CIs
- fixed commands that were using old docker repository - scality/cloudserver, and changed it to zenko/cloudserver
- added an environmental variable to docker run cloudserver without Orbit

We need this because we have questions coming in about cloudserver, it seems people are using it or want to use it, so docs need some love.

### CODE REVIEW

File: docs/DOCKER.rst [hunk 1]
```diff
@@ -96,11 +88,22 @@ new.host.com, for example, specify the endpoint with:
 
 .. code-block:: shell
 
-    $ docker run -d --name cloudserver -p 8000:8000 -e ENDPOINT=new.host.com scality/cloudserver
+    $ docker run -d --name cloudserver -p 8000:8000 -e ENDPOINT=new.host.com zenko/cloudserver
 
 .. note:: On Unix-like systems (Linux, OS X, etc.) edit /etc/hosts
    to associate 127.0.0.1 with new.host.com.
 
+REMOTE_MANAGEMENT_DISABLE=1
```
Comment (JianqinWang) [inline docs/DOCKER.rst]: keeping in line with the other env variable title formats, we could probably just title this section as `REMOTE_MANAGEMENT_DISABLE`

File: docs/DOCKER.rst [hunk 2]
```diff
@@ -96,11 +88,22 @@ new.host.com, for example, specify the endpoint with:
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CloudServer is a part of `Zenko <https://www.zenko.io/>`__. When you run CloudServer standalone it will still try to connect to Orbit by default (browser-based graphical user interface for the Zenko).
+
+Using this variable will default to accessKey1 and verySecretKey1 for credentials and disable the automatic Orbit management:
```
Comment (JianqinWang) [inline docs/DOCKER.rst]: maybe we could rephrase the opening:

Setting this variable to `1` will default to ...

File: docs/DOCKER.rst (hunk 1 above)
Comment (dashagurova) [reply docs/DOCKER.rst]: totally

**Comment by nicolas2bert** [comment]
Just a note (not sure if it should be added to this PR): The destination address of this section https://github.com/scality/cloudserver/blob/development/8.2/README.md#docker should be our Cloudserver Docker image.

**Review by JianqinWang — APPROVED** [review]

**Comment by dashagurova** [comment]
@bert-e approve

File: docs/DOCKER.rst [hunk 3]
```diff
@@ -96,11 +88,22 @@ new.host.com, for example, specify the endpoint with:
...
-    $ docker run -d --name cloudserver -p 8000:8000 -e ENDPOINT=new.host.com scality/cloudserver
+    $ docker run -d --name cloudserver -p 8000:8000 -e ENDPOINT=new.host.com zenko/cloudserver
 
 .. note:: On Unix-like systems (Linux, OS X, etc.) edit /etc/hosts
    to associate 127.0.0.1 with new.host.com.
 
+REMOTE_MANAGEMENT_DISABLE
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CloudServer is a part of `Zenko <https://www.zenko.io/>`__. When you run CloudServer standalone it will still try to connect to Orbit by default (browser-based graphical user interface for the Zenko).
```
Comment (rachedbenmustapha) [inline docs/DOCKER.rst]: "for the Zenko" sounds weird

**Review by rachedbenmustapha — CHANGES_REQUESTED** [review]

File: docs/DOCKER.rst (hunk 3 above)
Comment (dashagurova) [reply docs/DOCKER.rst]: I think so too

Comment (dashagurova) [reply docs/DOCKER.rst]: done

**Review by rachedbenmustapha — APPROVED** [review]
@dashagurova I assume you've tested the instructions manually - since these commands (with the embedded metadata server for example) are not something we do routinely ourselves or for our customers, there could be a regression uncaught by tests.

**Pull request merged**